### ISSUE TITLE
No filament jam detection in dual-extruder system

### ISSUE THREAD

**Issue by Drzet** [maocypher/Octoprint-Smart-Filament-Sensor#23]
I have had successful detections in single extruder prints but I can now confirm jams are not detected during dual extrusion print.
Obviously the plugin has no way to know which extruder is supposed to be extruding at any given time. 
Is there a plan to introduce g-code parsing for `T0` `T1` commands ?

**Comment by maocypher** [comment]
Yes, it is planned, but it takes some time, because I can only work on it in my free time
